ArrtSource/Rendering/ArrSession.cpp

```diff
@@ -225,6 +227,55 @@ void ArrSession::CheckEntityBoundsResult(RR::Bounds bounds)
 //////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////
 
+void ArrSession::AggregateStats()
+{
+    m_numFrames = m_frameStatsHistory.size();
+
+    if (m_numFrames == 0)
+    {
+        return;
+    }
+
+    // clean up saved stats
+    m_frameStats = RR::FrameStatistics{};
+
+    for (auto& frameStats : m_frameStatsHistory)
+    {
+        m_frameStats.LatencyPoseToReceive += frameStats.LatencyPoseToReceive;
+        m_frameStats.LatencyReceiveToPresent += frameStats.LatencyReceiveToPresent;
+        m_frameStats.LatencyPresentToDisplay += frameStats.LatencyPresentToDisplay;
+        m_frameStats.TimeSinceLastPresent += frameStats.TimeSinceLastPresent;
+
+        m_frameStats.VideoFrameReusedCount += frameStats.VideoFrameReusedCount > 0 ? 1 : 0;
```

Comment:
why not accumulating all?

Reply:
this value shows how many times a frame was reused so we convert it to a bit just to check whether that frame was reused.